### PR TITLE
add tests for fixme (only basic use)

### PR DESCRIPTION
**Pull request by wiobber**

### CODE REVIEW

File: _data/tagging-status.yml [hunk 1]
```diff
@@ -1533,11 +1533,11 @@
 
  - name: fixme
    type: package
-   status: unknown
+   status: partially-compatible
```
Comment (FrankMittelbach) [inline _data/tagging-status.yml:1536]: looking at the pdf structure generated I think this can (for now) be classified as compatible

**Review by FrankMittelbach — CHANGES_REQUESTED** [review]
suggest to classify as compatible

File: _data/tagging-status.yml (hunk 1 above)
Comment (wiobber) [reply _data/tagging-status.yml]: stats updated to 'compatible'

**Comment by mbertucci47** [comment]
I think the changes to tagging-status.yml got lost somewhere

**Comment by davidcarlisle** [comment]
@mbertucci47  that's odd I just looked at the commit history and not sure quite where that got dropped. so it should be `compatible` ? I can fix locally  but I wanted to check I understood the comment

**Comment by davidcarlisle** [comment]
@mbertucci47  0292cc1

**Pull request merged**